applications/machine_learning/CMakeLists.txt

```diff
@@ -34,6 +34,9 @@ set(mcuboot_CONF_FILE
 set(b0_CONF_FILE
   ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/b0_${CMAKE_BUILD_TYPE}.conf
   )
+set(hci_rpmsg_CONF_FILE
+  ${CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/child_image/hci_rpmsg_${CMAKE_BUILD_TYPE}.conf
```

Comment:
I think we could define path as:   `$CMAKE_CURRENT_SOURCE_DIR}/configuration/${BOARD}/hci_rpmsg_${CMAKE_BUILD_TYPE}.conf`. That would be more consistent with our way of defining configuration for bootloaders.

Reply:
No strong opinion but NCS users may be used to having `child_image` directory for other cores. The thing you describe are child images for same core, which are then merged into single hex. @pdunaj what is your opinion on this?

Reply:
Talked offline with @pdunaj, changed as proposed.